src/components/adaptive-cards/AdaptiveCard/AdaptiveCard.jsx

```diff
@@ -155,8 +161,15 @@ export default function AdaptiveCard({
     });
 
     setInputs(newInputs);
+    const result = !Object.values(newInputs).some((input) => input.error);
 
-    return Object.values(newInputs).some((input) => input.error);
+    if (!result) {
```

Comment:
move this in the Action.Submit.jsx file, in the submit logic there is a "if (validate()" check, add an "else" branch